src/config.cpp

```diff
@@ -183,6 +183,101 @@ struct Outputs_Element : JSON::Element {
   Config::Model::Decoder::Outputs& v_;
 };
 
+struct StringArray_Element : JSON::Element {
+  explicit StringArray_Element(std::vector<std::string>& v) : v_{v} {}
+
+  void OnString(std::string_view name, std::string_view value) override {
+    v_.push_back(std::string(value));
```

Comment:
Is the std::string needed?
```suggestion
    v_.push_back(value);
```

Reply:
```
error: no matching function for call to 'std::vector<std::__cxx11::basic_string<char> >::push_back(std::string_view&)
```

Conversion from std::string_view to std::string must be explicit.